Testing/Functional/Products/Tests/RemoveOutputFolders.ps1

```diff
@@ -0,0 +1,16 @@
+# The purpose of this script is to delete all the temp folders created by running ScubaGear.
+
+$thisDir = Get-Location
```

Comment:
Declaring the target directory as a parameter and defaulting to current location may make this more reusable.

Reply:
OBE.  This variable disappeared after moving the hashtable into a secret.  Done.

Reply:
I do not understand the resolved comment.  Variable still appears to be in baseline.

Reply:
I see what happened.  I had used this command in 2 different scripts, and I removed it from the other script.  Sorry, I got confused.

As far as changing this script to use an input parameter, no, I don't want to do that.  Eschew scope creep.  This script has a very specific purpose, it works great for that purpose.  I don't need it to be a more general-purpose script.  Similar to this discussion that we had for the other script, if you want it to be changed into something else, then please create a new issue and we can see if that's where the team wants to go.

Reply:
I am okay with defering changing.  We are currently modifying Scuba to be deployed from a package repo as a module. Part of this will modify where the reports are stored.  I expect the Functional Tests will be updated as well setup Scuba using the package repo - so the clean up here will need to accomodate (in this release).